.wp-env.json

```diff
@@ -3,8 +3,13 @@
 	"port": 8988,
 	"testsPort": 8989,
 	"plugins": [ "." ],
+	"mappings": {
+		"wp-content/themes": "./dev-env/themes"
+	},
 	"config": {
 		"WP_UPLOAD_MAX_FILESIZE": "128M",
-		"WP_MEMORY_LIMIT": "256M"
+		"WP_MEMORY_LIMIT": "256M",
+		"WP_ENVIRONMENT_TYPE": "development",
+		"WP_DEVELOPMENT_MODE": "all"
```

Comment:
It looks like this was committed accidentally.

Reply:
Actually, it wasn't an accident, though I suppose that should probably go in a separate PR.

The environment variables feel pretty necessary, when running the wp-env environment, to be running in development mode (to prevent pattern cacheing). And the mappings allows us to actually interact with the theme assets from an environment spun up from wp-env.  

Both important changes I think, so I'll open up a separate PR in which to discuss the merits.